decentraland/components/presentational/BuyLand.js

```diff
@@ -1,38 +1,60 @@
 import React from "react";
-import { StyleSheet, View } from "react-native";
+import { StyleSheet, View, Text } from "react-native";
 import { responsiveHeight } from "react-native-responsive-dimensions";
 import PropTypes from "prop-types";
 import LandForSale from "./LandForSale";
 import Button from "./Button";
 import Colors from "@constants/Colors";
 
 export default function BuyLand(props) {
-  const { landForSale, onBuy, waitingForAccountSetup } = props;
+  const { landForSale, onBuy, accountInfo } = props;
```

Comment:
Here is where we would change what is passed into this component https://github.com/tasitlabs/tasit/pull/228/files#r270520447

Reply:
Resolving this